src/main/java/org/broadinstitute/consent/http/resources/ElectionResource.java

```diff
@@ -115,4 +116,16 @@ public Response isDataSetElectionOpen(@Context UriInfo info) {
         }
     }
 
+    @GET
+    @Produces("application/json")
+    @PermitAll
+    @Path("/{electionId}/votes")
+    public Response describeVotesOnElection(@PathParam("electionId") Integer electionId) {
```

Comment:
Minor nit: We should add an auth user as a param even though we're not using it in any logic. 
A slightly bigger nit: I think we need to restrict this to admins, chairs, and members right now. This shouldn't be used in any researcher views yet, and when SOs become a thing, we can add them to the list.